doc/nrf/releases/release-notes-latest.rst

```diff
@@ -181,10 +181,39 @@ Zigbee
   * Reworked the :ref:`NCP sample <zigbee_ncp_sample>` to work with the simple gateway application.
   * Moved the `NCP Host documentation`_ from the `ZBOSS NCP Host`_ package to the same location as the `external ZBOSS development guide and API documentation`_.
 
+nRF Desktop
+-----------
+
+* Fixed boot reports on USB backend.
+* Use common modules from CAF library.
+* Minor bugfixes.
+
 Common
 ======
 
-There are no entries for this section yet.
+The following changes are relevant for all device families.
+
+Edge Impulse
+------------
+
+* Added the example application integrating edge impulse wrapper library with sensor sampling.
+  The code is located in :file:`ncs/nrf/applications/machine_learning`.
+* Sample reference to a public pre-trained model.
```

Comment:
Not sure what this is about. @MarekPieta ?

Reply:
That you don't have to train model yourself as there is a default model added to config.